server/src/main/java/org/spine3/server/command/CommandBus.java

```diff
@@ -130,19 +130,15 @@ public void unregister(CommandHandler handler) {
     /**
      * Verifies if the command can be posted to this {@code CommandBus}.
      *
-     * <p>The command can be posted if it has either dispatcher or handler registered with
+     * <p>A command can be posted if it has either dispatcher or handler registered with
```

Comment:
We also need to mention validation prior to describing returned results. Something like:

----
The command message must also satisfy validation constraints defined in its protobuf type.
----